core/src/tpu.rs

```diff
@@ -105,6 +109,13 @@ impl Tpu {
             bank_notification_sender,
             cluster_confirmed_slot_sender,
         );
+        let cluster_info_entries_listener = ClusterInfoEntriesListener::new(
+            exit,
+            cluster_info.clone(),
+            poh_recorder,
```

Comment:
If we're just using this to get a bank, I think passing `bank_forks` is probably better than `poh_recorder`